packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -318,24 +318,6 @@ export function selectOutput(
 	operator: WorkflowNode<any>,
 	selectedWorkflowOutputId: WorkflowOutput<any>['id']
 ) {
-	// Check if the current state existed previously in the outputs
-	let current = operator.outputs.find((output) => output.id === operator.active);
-	if (!current) {
-		// the current state was never saved in the outputs prior
-		current = {
-			id: uuidv4(),
-			type: '',
-			status: WorkflowPortStatus.NOT_CONNECTED,
-			isOptional: false
-		} as WorkflowOutput<any>;
-		operator.outputs.push(current);
-	}
-
-	// Update the current state within the outputs
-	current.state = cloneDeep(operator.state);
-	current.operatorStatus = operator.status;
-	current.timestamp = new Date();
-
```

Comment:
this wasn't needed anymore as I am using the `'append-output-port'` to add a new output

Reply:
Where do you save an output that was not `Run`?

Reply:
What would be an example of this?  For something like save model we are leveraging the 'append-output-port' as well

Reply:
When I made changes to the state, but have not clicked `Run`, then select a different output to visualize its state. I would expect to be able to get back to that last un-`run` state. So it needs to be saved even if it wasn't ran, therefore does not have an output.

Reply:
This will be solved in another PR.